### PR TITLE
pkp/pkp-lib#1457 urn locale changes

### PR DESCRIPTION
**Pull request by bozana**
S
https://github.com/pkp/ojs/pull/839/commits/24a63e1cbc7d95c5e61ab5c8a475dd4169cd93cd#diff-56
https://github.com/pkp/ojs/pull/839/commits/71720fa0c1b3ccb9b342ea28f3402167ab5032bc#diff-5
https://github.com/pkp/ojs/pull/897/commits/690bc8be57b7119333164b5f554ab7c6010c5c36#diff-15

### CODE REVIEW

**Comment by asmecher** [comment]
@mtub, mind looking at this one too?

File: plugins/pubIds/urn/locale/nl_NL/locale.xml [hunk 1]
```diff
@@ -12,65 +12,42 @@
...
-	<message key="plugins.pubIds.urn.manager.settings.form.urnArticleSuffixPatternRequired">Geef het URN achtervoegsel patroon voor artikelen.</message>
-	<message key="plugins.pubIds.urn.manager.settings.form.urnGalleySuffixPatternRequired">Geef het URN achtervoegsel patroon voor proeven.</message>
-	<message key="plugins.pubIds.urn.manager.settings.form.urnSuppFileSuffixPatternRequired">Geef het URN achtervoegsel patroon voor aanvullende bestanden.</message>
-	<message key="plugins.pubIds.urn.manager.settings.form.namespaceRequired">Kies een namespace.</message>
+	<message key="plugins.pubIds.urn.manager.settings.urnReassign">Wis URN's</message>
+	<message key="plugins.pubIds.urn.manager.settings.urnReassign.description">Als u de URN configuratie verandert heeft dat geen gevolgen voor reeds toegekende URN's. Nadat de gewijzigde URN configuratie is opgeslagen kunt u deze knop gebruiken om alle bestaande URN's te wissen om de nieuwe instellingen ook toe te passen op bestaande objecten.</message>
+	<message key="plugins.pubIds.urn.manager.settings.urnReassign.confirm">Weet u zeker dat u allle bestaande URN's wilt verwijderen?</message>
```
Comment (mtub) [inline plugins/pubIds/urn/locale/nl_NL/locale.xml]: this probably is a typo, s/allle/alle/, but I see that it has been in the file before :)

**Comment by mtub** [comment]
Looks good, @bozana

**Comment by asmecher** [comment]
Merged, thanks!

**Pull request merged**